src/llama-vocab.cpp

```diff
@@ -11,8 +12,11 @@
 #include <cstdarg>
 #include <cstring>
 #include <forward_list>
+#include <map>
 #include <queue>
+#include <set>
 #include <sstream>
+#include <unordered_map>
```

Comment:
I don't think `<sstream>` is used here.
```suggestion
#include <set>
#include <unordered_map>
```